### ISSUE TITLE
Unclear how to add accessibility label to paper-textarea

### ISSUE THREAD

**Issue by lesliana** [PolymerElements/paper-input#478]
For an example, see http://jsbin.com/hivovu/edit?html,output.  Turn on VoiceOver software and tab through the elements.

In the jsbin, if you add either aria-label or aria-labelledby to a normal html <textarea> element, VoiceOver software correctly reads the intended label in both cases.  However, if you add aria-label or aria-labelledby to the paper-textarea, they are ignored, and only the character count is read as the description.

From the source code, it looks like paper-textarea only has a protected _ariaLabelledby property and does not have any way to pass an aria-label to its child textarea.  What is the recommended way to add an accessibility label to a paper-textarea?

**Comment by notwaldorf** [comment]
Ah yes; we use the `aria-labelledby` attribute to take the contents of the `label` attribute and pass it on to the nested textarea, so I recommend using the `label` attribute for the Voice Over text. However, if you don't want that label to be showing, you can just hide it with a mixin:

```html
<style is="custom-style">
      paper-textarea.no-label {
        --paper-input-container-label: {
          display: none;
        };
      }
</style>

<paper-textarea label="description" class="no-label"
    always-float-label char-counter maxlength="500"></paper-textarea>
```

Here's a JSBin with an example: http://jsbin.com/fepepop/edit?html,output

Let me know if that works for you!

**Comment by lesliana** [comment]
That works great for me, thanks for the suggestion!

**Comment by notwaldorf** [comment]
Yay!

**Comment by hcarmona** [comment]
Can we re-open this issue? It looks like the textarea's aria-labelledby is populated with the id of the label, however, the label is not in the same shadow dom as the textarea.

```
<paper-input-container>
  <<shadow-root>>
    <label id="paper-input-label-1"></label>
    <iron-autogrow-textarea>
      <<shadow-root>>
        <textarea aria-labelledby="paper-input-label1"></textarea>
      <</shadow-root>>
    </iron-autogrow-textarea>
  <</shadow-root>>
</paper-input-container>
```

This means that screen readers are not finding the label. (Same is true for aria-describedby)

**Comment by johensantos** [comment]
I have the same problem of @hcarmona  and @umbertoand , if someone knows how to solve it I would be very grateful.
